lib/Conversion/StandardToHandshake/StandardToHandshake.cpp

```diff
@@ -1470,7 +1563,7 @@ struct HandshakePass
   }
   /// Return the dialect that must be loaded in the context before this pass.
   void getDependentDialects(::mlir::DialectRegistry &registry) const override {
-    registry.insert<HandshakeOpsDialect>();
+    registry.insert<HandshakeOpsDialect, mlir::AffineDialect>();
```

Comment:
You shouldn't need the Affine Dialect here, because this pass is not creating any Affine operations.